### DISCUSSION TITLE
Prometheus rules

### DISCUSSION THREAD

**Discussion by jr0dd** [fluxcd/flux2#1789] [Q&A] [answered]
I was curious if there were any Prometheus alerts and/or recording rules available. I found the grafana dashboards, but it would be nice to have some default rules as well. Unless the norm is to just use the notification-controller.

**Comment by allenporter** [comment, marked as answer]
For completeness, responding here too: I've used the `ReconciliationFailure` rule https://fluxcd.io/docs/guides/monitoring/#metrics with success.

**Reply by jr0dd** [reply]
> For completeness, responding here too: I've used the `ReconciliationFailure` rule https://fluxcd.io/docs/guides/monitoring/#metrics with success.

Thanks! I must have missed that in the docs. That one is definitely the most important one I want to be alerted for.
